modules/core/src/v2/coins/sol.ts

```diff
@@ -100,8 +99,35 @@ export class Sol extends BaseCoin {
   }
 
   // TODO (https://bitgoinc.atlassian.net/browse/STLX-10202)
-  async verifyTransaction(params: VerifyTransactionOptions): Promise<boolean> {
-    throw new MethodNotImplementedError('verifyTransaction method not implemented');
+  async verifyTransaction(params: VerifyTransactionOptions): Promise<any> {
+    let totalAmount = 0;
```

Comment:
Need to think what is we have a tx with amout 0. This will not work with 0 amount and always will be true.